nerfstudio/process_data/hloc_utils.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+from typing_extensions import Literal
+
+from nerfstudio.process_data.process_data_utils import CameraModel
+from hloc import extract_features, match_features, reconstruction, visualization, pairs_from_retrieval, pairs_from_retrieval, pairs_from_exhaustive
+
+from hloc.triangulation import (import_features, import_matches)
```

Comment:
These should be wrapped in a try/except and notify the user how to install if they are missing.

Reply:
ok, I'll look at it.